package.json

```diff
@@ -48,5 +50,12 @@
     "mode-watcher": "^0.1.2",
     "tailwind-merge": "^2.1.0",
     "tailwind-variants": "^0.1.18"
-  }
+  },
+  "overrides": {
+		"@sentry/cli": "2.22.2"
+	},
+	"optionalDependencies": {
+		"@sentry/cli-darwin": "2.22.2",
+		"@sentry/cli-linux-x64": "2.22.2"
+	}
```

Comment:
Required to fix an issue with Sentry on Vercel.

See here: https://github.com/getsentry/sentry-cli/issues/1862#issuecomment-1854091275